envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/SnapshotUpdater.kt

```diff
@@ -46,7 +61,12 @@ class SnapshotUpdater(
     }
 
     private fun updateSnapshotForGroup(group: Group, snapshot: Snapshot) {
-        val groupSnapshot = snapshotFactory.getSnapshotForGroup(group, snapshot)
-        cache.setSnapshot(group, groupSnapshot)
+        try {
+            val groupSnapshot = snapshotFactory.getSnapshotForGroup(group, snapshot)
+            cache.setSnapshot(group, groupSnapshot)
+        } catch (e: Throwable) {
+            meterRegistry.counter("snapshot-updater.services.${group.serviceName}.updates.errors").increment()
```

Comment:
Do we have anything other under `snapshot-updater.services`? If not then I suggest `snapshot-updater.updates.services.{...}.errors`, goes well with `snapshot-updater.updates.errors`.